.github/workflows/aro-hcp-dev-env-cd.yml

```diff
@@ -56,18 +56,30 @@
           with:
             azcliversion: latest
             inlineScript: |
+              REGION=westus3
               SC_RESOURCEGROUP="aro-hcp-dev"
               MC_RESOURCEGROUP="aro-hcp-dev-mc"
-              REGION=westus3
+              REGIONAL_RESOURCEGROUP="aro-hcp-dev-$REGION"
               cd dev-infrastructure/
 
-              az group create -g "${SC_RESOURCEGROUP}" -l "${REGION}" --tags persist=true
-              az group create -g "${MC_RESOURCEGROUP}" -l "${REGION}" --tags persist=true
+              az group create -g "${SC_RESOURCEGROUP}"       -l "${REGION}" --tags persist=true
+              az group create -g "${MC_RESOURCEGROUP}"       -l "${REGION}" --tags persist=true
+              az group create -g "${REGIONAL_RESOURCEGROUP}" -l "${REGION}" --tags persist=true
+
+              # DNS
+              az deployment group create \
+                --name "$(GITHUB_RUN_ID)" \
+                --resource-group $(REGIONAL_RESOURCEGROUP) \
+                --template-file templates/dev-dns.bicep \
+                --parameters \
+                  configurations/dev-dns.bicepparam \
+                --parameters \
+                  currentUserId=$(CURRENTUSER)
 
               # dev Azure Container Registry (ACR)
               az deployment group create \
                 --name "dev-acr-${GITHUB_RUN_ID}" \
-                --resource-group "${SC_RESOURCEGROUP}" \
```

Comment:
Moves ACR into regional resource group. 

Next steps would be to move this to a "global" resource group and leverage it across all regional deployments.